README.md

```diff
@@ -1,6 +1,6 @@
 # Vizbee Android Demo App
```

Comment:
Vizbee Continuity Integration Steps for your Android mobile app
Vizbee HomeSSO Integration Steps for your Android mobile app
No “2 demo app” headings!

Reply:
Also, soon we’ll have a requirement for a demo app demonstrating the simple sign in flow where mobile just sends the sign in details and TV simply completes the sign in with that info. Let’s think about that too after this one.

Reply:
Done